README.md

```diff
@@ -131,38 +180,57 @@ $ brew install carthage
 If your project does not have a Cartfile yet, use the `touch Cartfile` command in the root directory of your project. To install the IBM Watson Swift SDK using Carthage, add the following to your Cartfile.
 
 ```
-github "watson-developer-cloud/swift-sdk" ~> 4.0.1
+github "watson-developer-cloud/swift-sdk" ~> 4.0
```

Comment:
Same as with the pod installation instructions above, this version number shouldn't be hand edited.